stacker/lookups/handlers/dynamodb.py

```diff
@@ -0,0 +1,129 @@
+from botocore.exceptions import ClientError
+import re
+from stacker.session_cache import get_session
+
+from ...util import read_value_from_path
+
+TYPE_NAME = "dynamodb"
+
+
+def handler(value, **kwargs):
+    """Get a value from a dynamodb table
+
+    dynamodb field types should be in the following format:
+
+        [<region>:<tablename>@]<primarypartionkey>.<keyvalue>.<keyvalue>...
```

Comment:
I think the `[]` shouldn't include the `<tablename>@` right?  That's not optional?